specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-07-01-preview/appconfiguration.json

```diff
@@ -3075,6 +3075,17 @@
       "type": "string",
       "pattern": "^[^\\x00-\\x1F\\x7F]+$",
       "x-ms-parameter-location": "method"
+    },
```

Comment:
this looks the same as the one in common-types. why cant the one from common-types be used ?

Reply:
After discussing in the API Spec review channel, it was decided that the common type definition should be removed because no one can use it without getting linter warnings about not specifying a pattern. See #30343

Reply:
It's removed now.

Reply:
Resolving.